File: android_samples/02 App_basic_CallBackExamples/app/src/main/java/com/msnegi/callbackexamples/CallBackActivity.java
package com.msnegi.callbackexamples;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

public class CallBackActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_call_back);

        TextView txtHello = (TextView) findViewById(R.id.txtHello);
        txtHello.setText("An activity can't make any callback to calling activity. " +
                "An activity return control to calling activity through onActivityResult() function only.");
    }
}

File: android_samples_kotlin/12 CallbackExamples/app/src/main/java/com/msnegi/callbackexamples/CallBackActivity.kt
package com.msnegi.callbackexamples

import android.os.Bundle
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class CallBackActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_call_back)

        val txtHello = findViewById<View?>(R.id.txtHello) as TextView
        txtHello.setText(
            "An activity can't make any callback to calling activity. " +
                    "An activity return control to calling activity through onActivityResult() function only."
        )
    }
}
